string target_name
---
geometry_msgs/Point centroid_point
